### PR TITLE
Audit logging

### PR DESCRIPTION
**Pull request by sean-quinn**

### CODE REVIEW

File: source/sections/getting-started/logging.md [hunk 1]
```diff
@@ -20,4 +20,20 @@ kubectl -n kube-system patch pvc elasticsearch-data-es-client-1 --patch='{"spec"
...
+#### Audit 
+Any cluster can also have CKS host kernel-level audit logging enabled, which will be treated the same as application logs (sent to s3 on for the life of the cluster and indexed in elasticsearch for 5 days). Please submit a support ticket if you would like this feature enabled, make sure to take a look at the caveats below prior to creating your request. These audit logs include but are not limited to:
+* Kernel Parameter/Module Modifications
+* Mount Operations
+* Cron Scheduling
+* Sudoers, Passwd, User, Group, Password Database Changes
+* Network Environment Changes
+* Systemd Changes/Operations
+
+A full listing of example audit rules that are enabled by default can be found [here](https://github.com/Neo23x0/auditd/blob/master/audit.rules), however we have disabled some of these on CKS hosts due to not being applicable.
```
Comment (bartelsb) [inline source/sections/getting-started/logging.md:34]: I don't think we want to publish this information. Might be worth checking with the security team, but I think this opens up more questions than it answers.

File: source/sections/getting-started/logging.md [hunk 2]
```diff
@@ -20,4 +20,20 @@ kubectl -n kube-system patch pvc elasticsearch-data-es-client-1 --patch='{"spec"
...
\ No newline at end of file
+If the available burst balance is continually drained due to high logging throughput, then the Elasticsearch volumes will need to be overprovisioned such that burst balance is no longer a factor. EBS volumes with a capacity of 1TB or more do not have a burst balance, since they generate enough IOPS to replenish burst before it can be used. Expanding your elasticsearch-data-es-client PVCs to 1000Gi will eliminate the IOPS bottleneck from the system, and significantly increase the number of logs that can be handled by the logging stack.
+
+#### Audit 
+Any cluster can also have CKS host kernel-level audit logging enabled, which will be treated the same as application logs (sent to s3 on for the life of the cluster and indexed in elasticsearch for 5 days). Please submit a support ticket if you would like this feature enabled, make sure to take a look at the caveats below prior to creating your request. These audit logs include but are not limited to:
```
[the remark this answers is not in the document]
Comment (sean-quinn) [reply source/sections/getting-started/logging.md]: updated

File: source/sections/getting-started/logging.md (hunk 1 above)
Comment (sean-quinn) [reply source/sections/getting-started/logging.md]: removed

File: source/sections/getting-started/logging.md [hunk 3]
```diff
@@ -20,4 +20,20 @@ kubectl -n kube-system patch pvc elasticsearch-data-es-client-1 --patch='{"spec"
...
+* Kernel Parameter/Module Modifications
+* Mount Operations
+* Cron Scheduling
+* Sudoers, Passwd, User, Group, Password Database Changes
+* Network Environment Changes
+* Systemd Changes/Operations
+
+A full listing of example audit rules that are enabled by default can be found [here](https://github.com/Neo23x0/auditd/blob/master/audit.rules), however we have disabled some of these on CKS hosts due to not being applicable. 
+
+By default this feature is disabled, some caveats that should be acknowledged prior to requesting to enable:
```
[the remark this answers is not in the document]
Comment (sean-quinn) [reply source/sections/getting-started/logging.md]: updated

File: source/sections/getting-started/logging.md [hunk 4]
```diff
@@ -20,4 +20,20 @@ kubectl -n kube-system patch pvc elasticsearch-data-es-client-1 --patch='{"spec"
...
+* Significant increase in required size of logging volumes and increased likelihood of needing to increase logging volume storage across all nodes. If those volumes are not actively monitored and they fill up they can cause service interruption by preventing unrelated pods from starting properly.
```
[the remark this answers is not in the document]
Comment (sean-quinn) [reply source/sections/getting-started/logging.md]: updated for clarity

File: source/sections/getting-started/logging.md [hunk 5]
```diff
@@ -20,4 +20,18 @@ kubectl -n kube-system patch pvc elasticsearch-data-es-client-1 --patch='{"spec"
...
\ No newline at end of file
+If the available burst balance is continually drained due to high logging throughput, then the Elasticsearch volumes will need to be overprovisioned such that burst balance is no longer a factor. EBS volumes with a capacity of 1TB or more do not have a burst balance, since they generate enough IOPS to replenish burst before it can be used. Expanding your elasticsearch-data-es-client PVCs to 1000Gi will eliminate the IOPS bottleneck from the system, and significantly increase the number of logs that can be handled by the logging stack.
+
+#### Audit 
+Any cluster can also have CKS host kernel-level audit logging enabled, these los will be treated the same as application logs (sent to S3 on for the life of the cluster and indexed in Elasticsearch for 5 days). Please submit a support ticket if you would like this feature enabled, make sure to take a look at the caveats below prior to creating your request. These audit logs include but are not limited to:
```
Comment (bartelsb) [inline source/sections/getting-started/logging.md:26]: "los" should be "logs".

Comment (sean-quinn) [reply source/sections/getting-started/logging.md]: updated

**Pull request merged**